lua/acf/menu/items_cl/baseplates.lua

```diff
@@ -0,0 +1,70 @@
+local ACF        = ACF
+
+local gridMaterial = CreateMaterial("acf_bp_vis_spropgrid1", "VertexLitGeneric", {
```

Comment:
This still mentions SProps. It doesn't really matter that much, but it still should probably just get changed to something else.